handbook/people-ops/index.md

```diff
@@ -19,6 +19,7 @@
 - [Expense reports](expenses.md)
 - [Vacation/FTO](https://github.com/sourcegraph/Graphbook/blob/master/Holidays,%20FTO,%20and%20leaves%20of%20absence/Vacation%20and%20FTO.md)
 - [Holidays](holidays.md)
+- [Charities](charities.md)
```

Comment:
cc @mercadon not sure if there's a better place for this link.

Reply:
I think for now it's probably the best place, but I think if we named it "Donations" might make this a bit more clear for the index.

Reply:
https://github.com/sourcegraph/about/pull/1078